androgee/androgee.py

```diff
@@ -18,72 +32,94 @@
-            await ctx.send(file=image)
-        else:
-            ctx.send(f"!tempmute {member.mention} 300 being too rowdy")
-            message = f"{member.mention} was sprirzered by {ctx.message.author.mention}"
-            await ctx.send(message, file=image)
-    except commands.MissingRole:
-        image = get_image(ctx)
-        if member is None:
-            await ctx.send(file=image)
-        else:
-            message = f"{member.mention} was sprirzered by {ctx.message.author.mention}"
-            await ctx.send(message, file=image)
+def isbad(word: str) -> bool:
+    if word.lower() in swear_list:
+        return True
+    return False
 
 
-@bot.command(name="bonk")
-@commands.has_any_role(mod_role_name, mod_role_id)
-async def bonk(ctx, member: discord.Member = None):
-    try:
+class Androgee(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print(f"We is logged in as {self.bot.user}")
+
+    @commands.command(name="spray", aliases=["spritzered"])
+    async def spray(self, ctx, member: discord.Member = None):
+        if mod_role_id in [y.id for y in ctx.author.roles] and member is not None:
+            await member.send("Hey just a heads up you where to rowdy, tone it down")
+            await ctx.channel.clear(10)
         image = get_image(ctx)
         if member is None:
             await ctx.send(file=image)
         else:
-            ctx.send(f"!tempmute {member.mention} 300 being too rowdy")
-            message = f"{member.mention} was bonked by {ctx.message.author.mention}"
+            message = f"{member.mention} was sprirzered by {ctx.message.author.mention}"
             await ctx.send(message, file=image)
-    except commands.MissingRole:
+
+    @commands.command(name="bonk")
+    async def bonk(self, ctx, member: discord.Member = None):
+        if mod_role_id in [y.id for y in ctx.author.roles] and member is not None:
+            await member.send("Hey just a heads up you where to rowdy, tone it down")
+            await ctx.channel.purge(10)
         image = get_image(ctx)
         if member is None:
             await ctx.send(file=image)
         else:
             message = f"{member.mention} was bonked by {ctx.message.author.mention}"
             await ctx.send(message, file=image)
 
+        @commands.command(name="source")
+        async def source(self, ctx):
+            message = f"{ctx.author.mention} the source is at https://github.com/Egeeio/androgee"
+            await ctx.send(message)
 
-@bot.command(name="source")
-async def source(ctx):
-    message = (
-        f"{ctx.author.mention} the source is at https://github.com/Egeeio/androgee"
-    )
-    await ctx.send(message)
+    @commands.Cog.listener()
+    async def on_message(self, ctx):
+        check = await self.last_message(ctx, ctx.content)
+        if check:
+            await ctx.channel.send(f"<@!{mod_role_id}> spamer")
```

Comment:
This doesn't seem very useful. It just not-pings the mods and says "spammer" and that's it?

Reply:
I'm updating it to lock the channel where it thinks people are spamming